textdb/textdb-dataflow/pom.xml

```diff
@@ -41,6 +41,21 @@
             <artifactId>textdb-storage</artifactId>
             <version>${textdb.version}</version>
         </dependency>
+
+
+        <dependency>
+            <groupId>edu.stanford.nlp</groupId>
+            <artifactId>stanford-corenlp</artifactId>
+            <version>3.6.0</version>
+        </dependency>
+        <dependency>
+            <groupId>edu.stanford.nlp</groupId>
+            <artifactId>stanford-corenlp</artifactId>
+            <version>3.6.0</version>
+            <classifier>models</classifier>
+        </dependency>
+
+
         <dependency>
             <groupId>org.mockito</groupId>
```

Comment:
Do you think you can implement the test cases using Mockito?  @sandeepreddy602 and @rajesh9625 : suggestions?